userspace/sysdig/sysdig.cpp

```diff
@@ -1651,6 +1646,21 @@ sysdig_init_res sysdig_init(int argc, char **argv)
 			}
 		}
 
+#ifndef _WIN32
+	char* no_color = getenv("NO_COLOR");
+	if (isatty(fileno(stdout)) &&
+		(no_color != nullptr && strcmp(no_color, "1") != 0) && !no_color_flag)
```

Comment:
Minor nits (not going to block the merge):
* the NO_COLOR spec doesn't require `1`, just a non-empty string (ok, that's a pretty major one)
* do we want --color for forcing color anyway? the flag would have to be a tristate then (no color ever, color only when isatty(stdout), always color) -- I think the NO_COLOR spec suggests that too
* we could extract the isatty check to e.g. a static inline function:

```
static inline bool stdout_supports_color()
{
#ifdef _WIN32
    return false;
#else
    return isatty(fileno(stdout));
}
```

* the env check could influence the no_color_flag rather than have a separate check